agent/recognition/binarymatch.py

```
@@ -697,15 +881,93 @@ def _run_standalone(self, argv: CustomRecognition.AnalyzeArg, params: dict):
+            self._log_rescue(node, rescue)
+            winner = rescue.get("_winner")
+            if (winner is not None and rescue_cfg["mode"] == "active"
+                    and rescue.get("_decision") == "stable_hit"):
+                confirmed = self._rescue_confirm_full(
+                    hsv_np=hsv_np, winner=winner,
+                    area_range=(area_min, area_max),
+                    aspect_range=(asp_lo, asp_hi), gap_ratio=gap_ratio,
+                    min_conf=min_conf, rx=rx, ry=ry)
+                if confirmed is None:
+                    rescue["停因"] = "全区复核不一致"
+                    rescue["说明"] = "局部结论未能在整幅 ROI 复现，已 fail closed"
```

Ruff (0.16.0)

[warning] String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)